gensim/models/keyedvectors.py

```diff
@@ -519,9 +518,17 @@ def rank(self, key1, key2):
         """Rank of the distance of `key2` from `key1`, in relation to distances of all keys from `key1`."""
         return len(self.closer_than(key1, key2)) + 1
 
-    # backward compatibility; some would be annotated `@deprecated` if that stacked with @property/.setter
     @property
     def vectors_norm(self):
+        raise ValueError(
+            "The vectors_norm attribute became a get_normed_vectors() method in Gensim 4.0.0. "
+            "See https://github.com/RaRe-Technologies/gensim/wiki/Migrating-from-Gensim-3.x-to-4#init_sims"
+        )
+
+    def get_normed_vectors(self):
+        # TODO: what's the way for users to get from a matrix index (integer) to the
```

Comment:
@gojomo FYI

Reply:
Yes. The idea here was to fuse `unit_normalize_all()` with `get_normed_vectors()`: have a single function which returns a new KV instance, normalized.

I think that's cleaner than several in-situ state modifiers/getters, but it's true the use-case is hypothetical.

Reply:
> get_vector(key_or_int, use_norm=True) returns an individual normed vector, by key or raw integer slot

Hmm, I don't see that in the code. How does the "raw integer slot" option work?

Reply:
I removed the `int` option from the docstring, because I don't think it exists.